pkg/utils/config/go.mod

```diff
@@ -0,0 +1,5 @@
+module nri/nri-plugins/pkg/utils/config
+
+go 1.19
+
+require github.com/pelletier/go-toml/v2 v2.0.9
```

Comment:
nit: containerd itself is using go-toml v1 (v1.9.5). Wouldn't it be safer to use the same version.

Reply:
I'm not sure. I'm not sure if using v2 can cause us some issues when modifying the config. At least, when I tried to modify it with v2 and pass that config to containerd it was fine. And I hope containerd is would also switch to v2.

Reply:
Okay, fair enough.